apps/kahoot-it-clone/src/app/quiz/question/question.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, Input } from '@angular/core';
+import { FormGroup, FormArray, FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'kahoot-it-clone-question',
+  templateUrl: './question.component.html',
+  styleUrls: ['./question.component.css']
+})
+export class QuestionComponent {
+  @Input() questionForm: FormGroup;
+
+  selected = '';
+
+  times = [
+    { value: 5 },
+    { value: 10 },
+    { value: 20 },
+    { value: 30 },
+    { value: 60 },
+    { value: 90 },
+    { value: 120 },
+    { value: 240 }
+  ];
+
```

Comment:
This can just be an array of numbers. times = [5, 10, 20, 30...]. You could simplify it even further by doing something like times = Array.from(new Array(10)).map((_, i) => 5 * (i +1)) and then set the values to equal whatever you want them to be.